scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lga.yaml

```diff
@@ -209,3 +209,28 @@ ST_HLGA-10_2.5x2.5mm_P0.6mm:
   #pad_length_addition: 0.5
   #suffix: '_ClockwisePinNumbering'
   #include_suffix_in_3dpath: 'False'
+
+Broadcom_LGA-8_2.0x2.0mm:
+  device_type: 'LGA'
+  size_source: 'https://docs.broadcom.com/docs/AV02-4755EN'
+  library: 'OptoDevice'
+  manufacturer: 'Broadcom'
+  ipc_class: 'qfn_pull_back'
+  body_size_x:
+    nominal: 2
+    tolerance: 0.1
+  body_size_y:
+    nominal: 2
+    tolerance: 0.1
+  overall_height:
+    nominal: 0.34
+    tolerance: 0.1
+  lead_width:
+    nominal: 0.31
+    tolerance: 0.1
+  lead_len:
+    nominal: 0.35
+    tolerance: 0.1
```

Comment:
For the LGA-8, we need the lead outer-edge-to-package-edge dimension. It is 0.05mm (1mm - 0.775mm - 0.35 / 2). You can capture it like this:
```
lead_to_edge:
    nominal: 0.05
```

Many definitions are captured differently in this file so please check around. I'm not that familiar with the LGA one. I think Rene added all the definitions.